docs/contributing/data-handling-and-conversion.md

```diff
@@ -34,7 +34,7 @@ At the bottom of this documentation is a [Glossary section](#glossary), which ma
     - [Nested TypeList of TypeString and AWS List of String](#nested-typelist-of-typestring-and-aws-list-of-string)
     - [Nested TypeMap of TypeString and AWS Map of String](#nested-typemap-of-typestring-and-aws-map-of-string)
     - [Nested TypeSet of Resource and AWS List of Structure](#nested-typeset-of-resource-and-aws-list-of-structure)
-    - [Nested TypeList of TypeString and AWS List of String](#nested-typelist-of-typestring-and-aws-list-of-string-1)
+    - [Nested TypeSet of TypeString and AWS List of String](#nested-typeset-of-typestring-and-aws-list-of-string)
```

Comment:
just needs this heading `nested-typeset-of-typestring-and-aws-list-of-string`  to be updated in the docs below as well